src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/Pipeline.cs

```diff
@@ -0,0 +1,597 @@
+        /// <since_tizen> 8 </since_tizen>
+        public SourceNode GetSource(string name)
+        {
+            NNStreamer.CheckNNStreamerSupport();
+
+            /* Check the parameter */
+            if (string.IsNullOrEmpty(name))
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "Node name is invalid");
+
+            SourceNode node;
+
+            if (_nodeList.ContainsKey(name))
+            {
+                if (_nodeList[name].Type != NodeType.Source)
+                    throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, name + " is not a source node");
+
+                node = (SourceNode) _nodeList[name];
+            }
+            else
+            {
+                node = new SourceNode(name, this);
+                _nodeList.Add(name, node);
+            }
+
+            return node;
+        }
+
+        /// <summary>
+        /// Gets the valve node instance with given node name.
+        /// </summary>
+        /// <param name="name">The name of valve node</param>
+        /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public ValveNode GetValve(string name)
+        {
+            NNStreamer.CheckNNStreamerSupport();
+
+            /* Check the parameter */
+            if (string.IsNullOrEmpty(name))
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "Node name is invalid");
+
+            ValveNode node;
+
+            if (_nodeList.ContainsKey(name))
+            {
+                if (_nodeList[name].Type != NodeType.Valve)
+                    throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, name + " is not a valve node");
+
+                node = (ValveNode) _nodeList[name];
+            }
+            else
+            {
+                node = new ValveNode(name, this);
+                _nodeList.Add(name, node);
+            }
+
+            return node;
+        }
+
+        /// <summary>
+        /// Gets the switch node instance with given node name.
+        /// </summary>
+        /// <param name="name">The name of switch node.</param>
+        /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
+        /// <exception cref="NotSupportedException">Thrown when the feature is not supported.</exception>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public SwitchNode GetSwitch(string name)
```

Comment:
Method which returns non-void value must have exactly one `<returns>` section.

Reply:
Thank you, I added 'returns' section.